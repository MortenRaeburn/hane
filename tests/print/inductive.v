Inductive P (A : Set) : forall a : A, Set := C : forall a : A, P A a
    with Q (A : Set) : Set := Q1 : Q A | Q2 : Q A.
Print P.
Print C.
