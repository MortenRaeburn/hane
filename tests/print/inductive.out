Inductive P (Set) : forall['0] (Set) :=
    | C : forall['0] (((P) ('1)) ('0))
    with Q (Set) : Set :=
    | Q1 : (Q) ('0)
    | Q2 : (Q) ('0).
